Source/player.h

```diff
@@ -631,7 +631,7 @@ void ClrPlrPath(Player &player);
 bool PosOkPlayer(const Player &player, Point position);
 void MakePlrPath(Player &player, Point targetPosition, bool endspace);
 void CalcPlrStaff(Player &player);
-void CheckPlrSpell(bool isShiftHeld);
+void CheckPlrSpell(bool isShiftHeld, spell_id spellID = Players[MyPlayerId]._pRSpell, spell_type spellType = Players[MyPlayerId]._pRSplType);

```

Comment:
I did not expect it to be possible to set this complex of a default :o

```suggestion
void CheckPlrSpell(bool isShiftHeld, spell_id spellID = MyPlayer->_pRSpell, spell_type spellType = MyPlayer->_pRSplType);
```